docs/source/contributing/typings.rst

```diff
@@ -0,0 +1,103 @@
+==============
+Adding Typings
+==============
```

Comment:
Add information about [PEP-563](https://www.python.org/dev/peps/pep-0563/), since we dropped py3.6 that should be something we can recommend.
See [py3.7 changelog](https://docs.python.org/3/whatsnew/3.7.html#pep-563-postponed-evaluation-of-annotations)

Reply:
No this a PR for restructuring only I’ll do additions in a later PR which I’ll pull soon.